src/unk_0203E880.c

```diff
@@ -692,7 +692,7 @@ BOOL sub_0203F278 (u16 param0)
     }
 }
 
-BOOL sub_0203F28C (u16 param0)
+BOOL Script_IsDoubleBattle (u16 param0)
```

Comment:
This function looks like it can be documented and simplified:

```c
BOOL Script_IsTrainerDoubleBattle(u16 trainerID)
{
    // TRDATA_BATTLE_TYPE is from trainer_data.h
    // BATTLE_TYPE_SINGLES is from constants/battle.h
    return TrainerData_LoadParam(trainerID, TRDATA_BATTLE_TYPE) != BATTLE_TYPE_SINGLES;
}
```

If the simplification doesn't match, feel free to revert it back to its usual form.